idea_town/frontend/static-src/app/views/feedback-view.js

```diff
@@ -56,9 +60,9 @@ export default BaseView.extend({
     // Hide the feedback form right away & let the submission happen in the
     // background - user probably doesn't care about progress or confirmation
     this.animateRemove();
+    this.parentCb();
```

Comment:
Need to check if `cb` was actually passed as an option - it's not in the "Give feedback" button case, and so an error gets thrown and submission never happens.